pkg/graveler/committed/meta_range.go

```diff
@@ -54,8 +54,9 @@ type DiffIterator interface {
 
 // RangeDiff represents a change in Range
 type RangeDiff struct {
-	Type  graveler.DiffType
-	Range *Range
+	Type         graveler.DiffType
+	Range        *Range
+	LeftIdentity ID
```

Comment:
can you add a doc code saying when it is populated

Reply:
Added, Thanks